test/libyul/yulOptimizerTests/commonSubexpressionEliminator/trivial.yul

```diff
@@ -1,11 +1,16 @@
 {
-    let a := mul(1, codesize())
```

Comment:
I guess you pushed this accidently, right?

Reply:
haha yeah, I noticed the same. I will just remove it and fix the `range::actions` deprecation error.